packages/react-lowcode/src/codegen/generation/interfaces/naming.ts

```diff
@@ -0,0 +1,31 @@
+import { camalizeString } from "../../../strings/camel";
+
+export interface Naming {
+    getBrowseComponentName(entityName: string): string
+    getDetailComponentName(entityName: string): string
+    getRootFolderName(entityName: string): string
```

Comment:
what is the purpose of this method?

did you mean something like `getProjectSourceFolder()` ?